extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -638,6 +638,66 @@ describe('amp-analytics.visibility', () => {
       });
     });
 
+    it('"hidden" trigger should work with duration condition', () => {
```

Comment:
Interesting. I thought my example would mean: "fires when page goes to background and element is currently visible". Anyway, I think this is another edge case that we should document.